genai-perf/genai_perf/config/input/config_perf_analyzer.py

```diff
@@ -77,37 +76,63 @@ def parse(self, perf_analyzer: Dict[str, Any]) -> None:
                 self.stability_percentage = value
             elif key == "measurement_interval":
                 self.measurement_interval = value
-            elif key == "request_count":
-                self.request_count.parse(value)
+            elif key == "warmup_request_count":
+                self.warmup_request_count = value
+            elif key == "measurement":
+                self.measurement.parse(value)
             else:
                 raise ValueError(
                     f"User Config: {key} is not a valid perf_analyzer parameter"
                 )
 
+    ###########################################################################
+    # Infer Methods
+    ###########################################################################
```

Comment:
Is this header necessary? Seems like we only have one method under it.

Reply:
It's not necessary, but it matches (my) the style of all the other config_*.py files. We only have one method right now, but it's very possible we have more in the future.